### DISCUSSION TITLE
<Why my images are not being display on my website which I am hosting in github, I copied the path from github directly. It is my first time! Please HELP haha thanks>

### DISCUSSION THREAD

**Discussion by Anaclara-AS** [community/community#132147] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Here you can see where I am trying to place the logo.png. Also this is the link for the website.

<img width="1377" alt="Screenshot 2024-07-14 at 19 04 07" src="https://github.com/user-attachments/assets/943cd4cd-6f48-4886-a690-7de0d4f45a19">
[https://anaclara-as.github.io/mywebsite/myportfolio/index.html](url)

**Comment by catunderrain** [comment, marked as answer]
Bro, leave it just "assets/imgs/logo.png" instead of "portfolio/assets/imgs/logo.png" lol

![image](https://github.com/user-attachments/assets/f320c2cc-ba81-40a7-94a4-660d62755f9d)

Because, the index file and assets all live in portfolio that mean, your root PATH is portfolio/.. . So that if you wanna call any stuff inside the root PATH, in this case the logo.png, just call from assets folder : D For ex, in case there is an image file name cat.png lives in porfolio same as index file, just call the src path: src="cat.png", hope you got it bro : D

**Reply by Anaclara-AS** [reply]
Hello Bro, I am a Sis hahaha. Thank you so much. I am a super hyper beginner. I really appreciate that you not only gave me the solution but also explained the reason behind it. Thanks, Bro. @catunderrain

**Reply by catunderrain** [reply]
No problem my Sis, hope you get all well in your own way, GL!

**Comment by danialtavakoli** [comment]
It looks like you have uploaded an image showing the structure of your GitHub repository and your `index.html` file. Based on your description and the screenshot, you are trying to display an image on your website but it's not showing up.

The issue could be due to an incorrect path to the image in your `index.html` file. Let's go through the possible reasons and solutions.

### Checking the Path

1. **Relative Path:** Ensure that the path you are using to reference the image is correct. In your HTML, you have:
   ```html
   <img src="myportfolio/assets/imgs/logo.png" alt="Anaclara Alza logo represented by two letters A in black color">
   ```
   Make sure that this path correctly points to the `logo.png` image file. If your `index.html` file is located at the root of the `myportfolio` folder, the path should be:
   ```html
   <img src="assets/imgs/logo.png" alt="Anaclara Alza logo represented by two letters A in black color">
   ```

2. **Case Sensitivity:** GitHub Pages and many web servers are case-sensitive. Ensure that the file names and directory names match exactly with the ones in the repository. For example, `Imgs` should be `imgs` if that's how it's named in the directory.

### Correcting the Path in Your HTML

Here is how your HTML should look:
```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>My Portfolio</title>
</head>
<body onload="scrollToTop()">
    <header class="header-main">
        <div class="header-main-logo">
            <img src="assets/imgs/logo.png" alt="Anaclara Alza logo represented by two letters A in black color">
        </div>
        <nav class="header-main-nav">
            <ul>
                <li><a href="#">MY WORK</a></li>
                <li><a href="#">ABOUT ME</a></li>
                <li><a href="mailto:anaclara.uxdesigner@hotmail.com?subject=Consultation%20Inquiry">CONTACT</a></li>
            </ul>
        </nav>
        <div class="waves">
            <div class="wave"></div>
            <div class="wave"></div>
            <div class="wave"></div>
            <div class="wave"></div>
        </div>
    </header>
    <main>
        <div class="container">
            <section class="intro">
                <div class="content">
                    <div class="typewriter-container">
                        <h1 class="typewriter">Hello! I am Anaclara Alza.</h1>
                    </div>
                    <div class="subtitle-container">
                        <p class="subtitle">A UX/UI designer who merges technology and creativity,</p>
                        <p class="subtitle">skilled in transforming engineering hardware processes into intuitive software solutions.</p>
                    </div>
                </div>
            </section>
        </div>
    </main>
</body>
</html>
```

### Additional Steps

1. **Clear Browser Cache:** Sometimes, browsers cache resources and changes might not appear immediately. Clear your browser cache or try accessing the site in an incognito window.

2. **Ensure Deployment:** After making changes, ensure you commit and push these changes to your GitHub repository. Wait for a few minutes for GitHub Pages to update.

3. **Check GitHub Pages Settings:** Ensure that GitHub Pages is correctly set up to serve the `index.html` file from the right branch and directory.

If after checking the path and ensuring everything is correctly set up you still face issues, please provide the URL of your GitHub Pages site for further assistance.

**Reply by Anaclara-AS** [reply]
Thank you so much for the super clear explanation! I appreciate it!

**Reply by danialtavakoli** [reply]
Great! If you found my response helpful, you can mark it as a correct answer to support me. Happy coding and good luck with your project! 👍

**Comment by LiteBrite82** [comment]
@Anaclara-AS 
Last thing to get your page deployed. Be sure to place your code in the root directory of the repo. (Take everything out of `myportfolio` and put in in the main directory)
